src/components/Layout.js

```diff
@@ -93,6 +97,11 @@ const Layout = (props) => {
         document.getElementById(idTag[1]).scrollIntoView(false)
       }
     }
+
+    // TODO: Remove Feb 9 2022
+    props.path.includes("/upgrades/") || props.path.includes("/staking/")
```

Comment:
Why not just set a variable here vs. component state?

Reply:
Yeah, I'd say if it's not dismissible, we don't really need to track any state. I can patch this.

Reply:
Makes sense. Thanks for addressing @wackerow.